### DISCUSSION TITLE
zh/guide/drivers/thunder

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#26] [Announcements]
# zh/guide/drivers/thunder

AList v3的文档

https://alist.nn.ci/zh/guide/drivers/thunder.html

**Comment by Corpama** [comment]
ThunderExpert配置好后，在aria2中下载时只能用获取到的UserAgent，而不能使用教程中那个DownUserAgent。而且当使用获取到的UserAgent进行下载时有限速，请问已经没有不限速的方法了吗

**Reply by Corpama** [reply]
提醒一下，如果是用的mac迅雷获取的参数，就会出现前面描述的限速，pc获取的不会有问题

**Comment by zzerosama** [comment]
新版改成横着卡片形式的了，不好复制那个链接啊

**Comment by vpei** [comment]
方法已经过时，复制打开链接后，把浏览器设置成手机一样的长方形，这样可以拉动下面滑动的图片，拼好图后，按以前的方法，会查看到内容Failed to launch 'xlaccsdk01://xunlei.com/callback?state=harbor&state=undefined&captcha_token=ckXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX&expires_in=558' because the scheme does not have a registered handler.
复制captcha_token的内容就能正常使用。

**Comment by ZimaKlein** [comment]
迅雷不显示parent_id

**Reply by QINSSTRIVE** [reply]
迅雷位于根目录下看不到parent_id，随便打开一个目录再试试应该就能找到了

**Reply by Terry0414** [reply]
但随便打开的那个目录就是该目录的id，而根目录的id始终没有啊

**Comment by yx10010** [comment]
抓包工具无法获取专家版参数，普通版用账户密码登录挂载成功，但是本地代理播放会卡顿，而且无法下载，弃坑，还是阿里靠谱

**Reply by drw2005** [reply]
确实，看视频卡顿，用处不大了。

**Reply by yibaogang** [reply]
今天我试了一下，不卡了！视频秒播，IDM下载试了下，能干到6M/S

**Comment by youtehub** [comment]
迅雷网盘要全速下载的话，要开超级会员，我就是开了超级会员后才支持全速下载的，否则会被限制限制速度，目前最新版本中，使用 迅雷专家版 时，只填写用户名，密码，验证码可以实现全速下载，无需填写入其他东西

**Reply by Hawkli0777** [reply]
我开了超级会员后手贱又多开了一个月白金，本来不限速，突然限速了

**Comment by FopFat** [comment]
迅雷网盘怎么才能分享指定的目录?而不是整个网盘。

**Reply by FopFat** [reply]
搞定了.在网盘页面,按F12 ,选择网络, 然后点击进入网盘你想挂载的目录,在控制台里网络那里会有一个文件: "iles?limit="的文件,点开之后,右侧会有 :"请求url:xxxxxx,然后后面会有一个100&parent_id=,  复制 id= 之后到&filters=之前的数据,就是你想挂在目录的ID"

**Comment by PaxonHuang** [comment]
最后一步短信验证码resource_exhausted报错不通过，有大佬遇到过吗？😢

**Reply by mapleshadow** [reply]
同样的问题。。。。

**Reply by ouclei** [reply]
每天10次申请短信验证，多了没有，只能等第二天

**Comment by Ohcecho** [comment]
在迅雷根目录下找不到parent id

**Reply by Ohcecho** [reply]
先点一下根目录下的其他文件夹，再刷新好像就可以了，好像会出现第三个files，里面有parent_id

**Comment by hzc54188** [comment]
这个难度太高

**Comment by yuanbao7353** [comment]
迅雷工作状态work，但是打开显示：failed get objs: failed to list objs: ErrorCode: 5 ,Error: file_not_found ,ErrorDescription: 文件或文件夹不存在，请检查确认

**Comment by IAdun** [comment]
挂载迅雷云盘后使用正常，电脑关机再开机，启用后迅雷云盘会掉线

**Comment by drw2005** [comment]
挂载的迅雷云盘普通版，但是看视频卡顿，看十来秒就缓冲一会，基本没啥用啊。

**Reply by kuko7** [reply]
我也是 你现在解决了吗

**Comment by fgjg123000** [comment]
迅雷挂载时报错 Failed create storage in database: UNIQUE constraint failed: x_storages.mount_path 这个怎么解决呀

**Comment by yibaogang** [comment]
迅雷云盘提速了，2个月前播放视频都很卡，今天我试了一下，可以秒播了！ 这爽的一批了呀，能离线下载，又能Alist 播放！

**Reply by Charlotteanty** [reply]
不能吧，不管是哪种方式挂载的迅雷云盘，都卡得很，完全不能在线看

**Reply by ouclei** [reply]
用potplayer还行

**Reply by CATBRO-HUI** [reply]
可能要会员吧？

**Comment by Charlotteanty** [comment]
高阶版挂载后，开启了下面的use video url，还是不能播放视频，显示不能连接到服务，这是哪里有问题吗

**Reply by Charlotteanty** [reply]
似乎是http和https的问题，可以播放，但不能挂载

**Comment by ccwctv** [comment]
无法播放最新咒术回战第二季，猜测应该是版权的问题迅雷官方限制了？？？
但是鬼灭之刃刀匠村篇却能播放，迅雷这操作挺迷的

**Comment by mapleshadow** [comment]
最新情况！
按照目前的教程，只需要输入账号、密码、根目录ID（只能选择某一个文件夹（例如我的转存）作为根目录）三个参数就行了，不需要那个验证的什么参数了！

**Comment by tintin814** [comment]
我只能透過微信登錄迅雷雲盤，無法設定手機號及密碼。請問該如何設置掛載？

**Comment by zfhxi** [comment]
要想两人在线看视频是不是得充会员才行（感觉非会员晚上会限速），开白金足够用吗？还是得开超级会员？

**Reply by CrazyJasonwell** [reply]
迅雷好像又不能用，按上面操作，不弹need verify 窗口信息了，大家什么情况

**Comment by qwe159357890** [comment]
今早登录发现报错:
Failed init storage: ErrorCode: 4002 ,Error: captcha_invalid ,ErrorDescription: meta.username expect 13****078, but got map[phone_number:+86 13****078 result:accept], please check captcha init params
请问是我自己问题还是其他大佬也有遇到这情况的?

**Reply by Johnson1618** [reply]
我也是这个问题，token明明是对的啊，但就是挂不上，显示4002错误

**Reply by qwe159357890** [reply]
而且像上面大佬说的 不弹need verify 窗口信息了

**Reply by Johnson1618** [reply]
现在也不知道解决办法，验证码发太多次了，迅雷提示操作频繁发不了验证码了

**Reply by lbqidu** [reply]
我也是这样，突然坏了，等更新吧

**Reply by cxy9520** [reply]
我的也是 前天坏的一直没有好，改密码，验证码 试了好几次都不行

**Reply by Johnson1618** [reply]
专家版还可以用

**Reply by yaomingshiwo** [reply]
同样这个问题，采用电话号码登录后，能够接受验证信息，但是填写CaptchaToken后还是显示ErrorCode: 4002 ,Error: captcha_invalid

**Reply by leoneeec** [reply]
在手机号前面加上+86就好了

**Reply by Johnson1618** [reply]
大佬牛逼，加了+86就行了，验证码都不用填了，恢复正常，谢谢大佬

**Reply by lbqidu** [reply]
确实有效，感谢！

**Reply by mtyuri** [reply]
好家伙，确实+86就行了。之前不行用专业版上的，有点麻烦。

**Comment by hiosh** [comment]
图2中的第一个链接怎么得到的？我先填了用户名和密码，验证码空着，然后提交，提交之后没有看到很长一串的网址啊。

**Comment by hiosh** [comment]
我得到的错误提示是这样的：
Failed init storage: ErrorCode: 4022 ,Error: invalid_account_or_password ,ErrorDescription: verification failed

**Reply by hiosh** [reply]
仔细看了，是密码输错，但我输对密码后中，又有了新的问题，代码4002，说是我手机号在验证的时候多了个+86，这又咋弄啊

**Reply by hiosh** [reply]
嘿嘿，重试成功了，在alist里登录，不要用手机号，改成邮箱登录，就可以了~~

**Reply by qwe159357890** [reply]
迅雷还能邮箱注册?

**Reply by hiosh** [reply]
可以，在i.xunlei.com里看看

**Reply by DIM0x** [reply]
看了一下没有邮箱设置啊…………

另外我看楼下有说在手机号前面输+86就成功了，我昨天这么搞的还是显示错误，今天突然就好用了……

**Reply by hiosh** [reply]
上面文档更新了，说是+86写在前面可以，我没试过，我现在还是用邮箱登的

**Reply by 14185638** [reply]
能否发一下邮箱注册的入口呢，找了一圈没找到

**Comment by hiosh** [comment]
i.xunlei.com
账号中心里设置


 佳
186 1630 8800
***@***.***
________________________________
发件人: qwe159357890 ***@***.***>
发送时间: 2023年10月2日 上午 7:49
收件人: alist-org/docs ***@***.***>
抄送: hio ***@***.***>; Comment ***@***.***>
主题: Re: [alist-org/docs] zh/guide/drivers/thunder (Discussion #26)


迅雷还能邮箱注册?

―

**Reply by hiosh** [reply]
管理员能把我这条回复删掉吗。。。。

**Comment by bmy888888** [comment]
普通迅雷版 我的怎么提示这个呢？？ErrorCode: 4022 ,Error: invalid_account_or_password ,ErrorDescription: invalid account or password

**Comment by cxy9520** [comment]
Failed load storage: failed init storage: ErrorCode: 4002 ,Error: captcha_invalid ,ErrorDescription: client_id:"Xp6vsxz_7IYVw2BB" meta:{key:"phone_number" value:"+86 159****5207"} meta:{key:"result" value:"review"} result:review
现在又不能用了 这又是什么报错

**Reply by ck981888399** [reply]
我也出现了，我猜是之前验证码的那个token无效了，但是没有解决办法啊

**Reply by tangwenlongNO1** [reply]
解决了吗

**Reply by CFSB** [reply]
Failed init storage: ErrorCode: 4002 ,Error: captcha_invalid ,ErrorDescription: client_id:"Xp6vsxz_7IYVw2BB" meta:{key:"phone_number" value:"+86 18937927721"} meta:{key:"result" value:"review"} result:review  我前几天按着教程成功了，但是才用3天就这样了

**Reply by tetsealeo** [reply]
应该是token过期了，试试用迅雷专业版，时间久一点

**Comment by lhw174780322** [comment]
求教我的版本v3.29.1挂载迅雷普通，没有获取验证码那部，就是直接添加完网盘就work了。但是在alist网盘端读不了迅雷云盘端的视频，显示加载失败。infuse可以，就是卡没阿里好用，是怎么回事？你们也有类似情况吗

**Comment by ChingyimLee** [comment]
使用用户名密码登陆，专家版速度变得超级慢，RefreshToken速度正常，这种是什么情况，RefreshToken总顶号，每次都得重新抓，比较麻烦。

**Comment by jzy20011128** [comment]
在线看视频看不了有人知道怎么解决吗

**Comment by Guan-feng** [comment]
我使用普通模式配置好了，也是work状态了，但就是报错。请哪位大神告诉我正确的处理办法？
failed get objs: failed to list objs: ErrorCode: 5 ,Error: file_not_found ,ErrorDescription: 文件或文件夹不存在，请检查确认
报上面这个错误。

**Comment by eutopiazen** [comment]
请问是迅雷限制速度了吗，看视频一卡一卡的

**Comment by saikishi** [comment]
出现了新的报错：Failed init storage: Post "https://xluser-ssl.xunlei.com/v1/shield/captcha/init": dial tcp: lookup xluser-ssl.xunlei.com on ***********:53: no such host，无论专家版还是普通都报

**Comment by NERO-Tsui** [comment]
ErrorCode: 4002 ,Error: captcha_invalid ,ErrorDescription: client_id:"Xp*****_*********B" meta:{key:"phone_number" value:"+86 1**********"} meta:{key:"result" value:"review"} result:review 怎么搞？

**Reply by EricCorleone** [reply]
一样的报错

**Reply by lxmqsjy** [reply]
我也是 不知道该怎么弄

**Reply by CFSB** [reply]
Failed init storage: ErrorCode: 4002 ,Error: captcha_invalid ,ErrorDescription: client_id:"Xp6vsxz_7IYVw2BB" meta:{key:"phone_number" value:"+86 "} meta:{key:"result" value:"review"} result:review 我前几天按着教程成功了，但是才用3天就这样了

**Reply by CFSB** [reply]
手机号加空格

**Comment by eutopiazen** [comment]
挂在了以后除了 infuse 能流畅播放，vidhub 和其他的都一卡一卡的
，不知道什么原因
Eric Wang ***@***.***>于2024年3月19日 周二11:16写道：

**Comment by gohaifjwja** [comment]
有提示关于手机号错误的去迅雷安全中心看一下自己的数字账号，用这个登陆成功了

**Reply by lxmqsjy** [reply]
3q 可用！

**Reply by yaomingshiwo** [reply]
使用数字账号后，可能会出现需要短信验证，验证的话按教程中的#CaptchaToken 第二张图片操作就行。edge浏览器稍微有不同，第八步CaptchaToken的数值在负载标签页中。

**Reply by CATBRO-HUI** [reply]
谢谢了，可以用了！！！

**Comment by ArmyDreamers** [comment]
用户名填迅雷UID成功了

但是为何我第一个迅雷账号成功了

添加第二个账号时 验证码 界面
显示的还是第一个账号的手机号呢?

**Comment by eutopiazen** [comment]
请问有没有挂在 infuse 和 vidhub 上用的，为什么 infuse 不卡，vidhub 卡啊

ArmyDreamers ***@***.***>于2024年4月5日 周五11:06写道：

**Comment by eutopiazen** [comment]
迅雷开了超会，但是 vidhub 播放时候播放几秒就会卡一下，infuse 就不会

猫哥 ***@***.***>于2024年4月9日 周二09:12写道：

**Reply by ChingyimLee** [reply]
infuse也卡。。。2000兆电信宽带

**Comment by ChingyimLee** [comment]
现在的迅雷专家版已经慢得可以了，要不是我充会员充到27年，真直接放弃了。

**Reply by tetsealeo** [reply]
ErrorCode: 7 ,Error: permission_denied, ErrorDescription: [Danger], Please Do Not Save Your client_secret in brower, it is NOT SAFE. 有没有大佬知道这个怎么解决？

**Reply by tetsealeo** [reply]
不好意思，回复错了

**Comment by tetsealeo** [comment]
ErrorCode: 7 ,Error: permission_denied, ErrorDescription: [Danger], Please Do Not Save Your client_secret in brower, it is NOT SAFE. 有没有大佬知道这个怎么解决？

**Comment by CFSB** [comment]
兄弟们，我的问题解决了！+8613722223333  换成 +86 13722223333  加空格！！！
Failed init storage: ErrorCode: 4002 ,Error: captcha_invalid ,ErrorDescription: client_id:"Xp6vsxz_7IYVw2BB" meta:{key:"phone_number" value:"+86 1"} meta:{key:"result" value:"review"} result:review
是看知乎找到的方法，本来我3天前用+8613722223333这样填写，普通版的流程走是没问题的，但是今天想继续同步到迅雷云盘发现里面都是空的，一看alist果然报错了，重新添加也不行，后来加了空格可以了。顺便说一下，不用流程里那么麻烦，等获取到验证码后直接把数字验证码填进去就ok了

**Comment by tanxy619** [comment]
macos，alist挂载迅雷网盘，本身是超级会员，点进去会提示“Request failed with status code 500“，前一天还用的好好的，有什么解决办法吗？Alist桌面版提示：ERRO[2024-05-05 23:45:31] failed get enabled storages: Post "https://xluser-ssl.xunlei.com/v1/shield/captcha/init": tls: first record does not look like a TLS handshake

**Comment by tanxy619** [comment]
谢谢解答！怎么样才能更新token呢？专业版好像很难设置😂
---- 回复的原邮件 ----
发件人 Tatsea ***@***.***> 日期 2024年05月09日 19:16 收件人 ***@***.***> 抄送至 ***@***.***>***@***.***> 主题 Re: [alist-org/docs] zh/guide/drivers/thunder (Discussion #26) 
应该是token过期了，试试用迅雷专业版，时间久一点

**Comment by tetsealeo** [comment]
下载Fiddler Web
Debugger，去抓迅雷客户端或网页端的包，在包里面拿到相应的参数，上面是有教程的，关于抓包软件的设置，去b站搜一下就行了，下载的话去官网
https://www.telerik.com/fiddler/fiddler-classic下比较好，因为新，缺点就是没有中文，想下的话就下Fiddle
Classic，也就这个是免费的。也可以在这里https://github.com/gabrielxvx/zh-fiddler
下载汉化版，但原作者已经弃坑了，最后一次更新还是2020年

tanxy619 ***@***.***> 于2024年5月9日周四 19:18写道：

**Comment by icetech233** [comment]
windows版本是 迅雷几啊

**Comment by panchuangye** [comment]
登录报错：
Failed init storage: need verify: <a target="_blank" href="https://i.xunlei.com/file/2s2or4vyf6jw2zso9u5/shield/captcha/v2/vertifyPhone.html?xxxxxxxxxxxxxxxxx>Click Here</a>
解决办法，复制need verify：后面<a 到</a>里面的内容，桌面新建一个文本文件把内容粘贴进去然后后缀名txt改成html,然后双击运行就会出来短信验证码页面，获取验证码后再返回登录就可以了

**Comment by RX78-3** [comment]
请问可以在迅雷中添加302模式吗

**Comment by eutopiazen** [comment]
专家模式里可以，但是很明显限速

RX78-3 ***@***.***>于2024年7月5日 周五15:57写道：

**Comment by youxdd** [comment]
迅雷专家版 太牛逼了 4k杜比 无压力, 80Mb速度, 搭配infuse 完美, 资源也好找, 不像阿里动不动就删,

**Reply by RX78-3** [reply]
老哥你咋抓到包的啊，我试了几天，网页版和PC端都试过。经常挂着等半个小时一个小时的就是抓不到那个包

**Reply by youxdd** [reply]
抓包工具有个过滤 , 只看需要的地址 就好找了

**Comment by youxdd** [comment]
安装个虚拟机抓包 不影响自己电脑登陆状态

**Reply by ChingyimLee** [reply]
不会影响登录状态么？我这经常互相顶

**Reply by ChingyimLee** [reply]
> 安装个虚拟机抓包 不影响自己电脑登陆状态

再就是，我这经常卡，是有啥操作不对么?

**Reply by youxdd** [reply]
我是mac电脑 我安装了windows虚拟机里面安装迅雷再抓包 没有影响mac的登陆状态，我用的推荐登陆方式 刷新token和验证码签名，我开的超级会员 没限速 80Mb

**Reply by ChingyimLee** [reply]
我懂了，windows和mac确实不会互相顶，甚至群晖都不会顶，但是windows和alist就会互相顶，80Mb，好像不足以看4K以上的。。。4K至少要100Mb吧，阿里云能到300多Mb。

**Reply by youxdd** [reply]
80够了 我一个电影30G 都不卡的 ，把那个下载url选项勾起 反正有几个选项都勾上

**Comment by eutopiazen** [comment]
我也是非常卡

ChingyimLee ***@***.***>于2024年7月24日 周三17:44写道：

**Reply by youxdd** [reply]
我用infuse看 4k 不卡

**Reply by ChingyimLee** [reply]
> 我用infuse看 4k 不卡

试试原盘或者remux的，我的迅雷专家版超过30g的视频绝对卡。

**Comment by youxdd** [comment]
那我没试过

ChingyimLee ***@***.***>于2024年7月24日 周三17:57写道：

**Comment by Zhangdata** [comment]
Failed init storage: ErrorCode: 4002 ,Error: captcha_invalid ,ErrorDescription: meta.username expect 18039991506, but got map[phone_number:+86 18039991506 result:accept], please check captcha init params
保存的时候报错，有谁知道怎么解决？

**Reply by Zhangdata** [reply]
我这个问题知道怎么解决了，原来手机号没有些+86，加上后就可以了（注意！！86后边要加个空格）

**Comment by foxdaiwx** [comment]
验证码，在第一次空白保存的时候，直接点击【保存】，然后出错，将手机号加上前缀+86后再【保存】后即完成

**Reply by Blue0127** [reply]
按照这个可以了

**Comment by scllz** [comment]
Client secret这个参数抓不到，windows下浏览器登陆找不到，手机登录抓包也看不到，那位介绍一下经验？

**Reply by ZZZ-zero** [reply]
PC下客户端抓包，教程里明确说了 Client secret 不存在于web端

**Reply by scllz** [reply]
那个说明我看到了，可是安卓手机网页登陆迅雷云盘，手机设置网络代理使用pc下的fiddler classic抓包，别的都有，迅雷专家版，迅雷浏览器专家版都抓不到Client secret，我还将手机上的云盘退去再进去也是没有用，因为我装了一个alist在手机上，抓不到头疼死了，那位成功的介绍一下，多谢了

**Comment by Louaq** [comment]
看视频卡顿。放弃了
